l10n/en/firefox/features/picture-in-picture.ftl

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/features/picture-in-picture/
+
+# HTML page title
+features-pip-firefox-picture-in-picture-get-more = { -brand-name-firefox } Picture-in-Picture: Get more done with pop-out videos
```

Comment:
Does it  make sense to make `Picture-in-Picture` a variable?

Reply:
I went and reviewed other feature FTL files. We don't make feature names a variable anywhere else. Please disregard.

Reply:
Yeah, I thought about that, but I don't think we treat it as a brand name. Looks like it's translated in Firefox. It's "Bild-im-Bild" in German and "Incrustation vidéo" in French! 🦀 🦞 🦐